Framework/Algorithms/src/PolarizationEfficiencyCor.cpp

```diff
@@ -210,7 +258,7 @@ void PolarizationEfficiencyCor::checkFredrikzeProperties() const {
   if (!isDefault(Prop::FLIPPERS)) {
     throw std::invalid_argument("Property Flippers cannot be used with the Fredrikze method.");
   }
-  if (!isDefault(Prop::SPIN_STATES)) {
+  if (!isDefault(Prop::OUTPUT_WILDES_SPIN_STATES)) {
     throw std::invalid_argument("Property SpinStates cannot be used with the Fredrikze method.");
```

Comment:
I think this error message needs to say "Property SpinStatesOutWildes cannot be used with the Fredrikze method." now that the Wildes spin states property has been renamed.